src/components/CheckIn/CheckInItemsWrapper.js

```diff
@@ -76,6 +83,32 @@ class CheckInItemsWrapper extends Component {
       .then(this.fetchItems);
   }
 
+  checkInItem = (values) => {
+    const { mutator, location } = this.props;
+
+    mutator.ORDER_PIECES.POST(values)
+      .then((piece) => {
+        this.callout.current.sendCallout({
+          type: 'success',
+          message: <FormattedMessage id="ui-orders.checkIn.addPiece.success" />,
+        });
+
+        return piece;
+      })
+      .then(piece => {
+        const item = getMixedPieceAndItem(piece);
```

Comment:
what is the point to call it passing in only a piece param?

Reply:
Removed from here and updated this function